test/unit/test-custom-element.js

```diff
@@ -1040,21 +1041,6 @@ describes.realWin('CustomElement', {amp: true}, (env) => {
         });
       });
 
-      it('StubElement - layoutCallback should fail before attach', () => {
```

Comment:
Just making sure: removing because StubElement is gone?

Reply:
No. `buildInternal` now waits until it can actually build. However, the places where `buildInternal` can be called is now significantly restricted (that was one of my previous PRs) and that's why there's a public-ish `build()` method. Basically this situation can no longer happen and even if it does, it'd still do the right thing.